app/src/main/java/com/asfoundation/wallet/billing/adyen/AdyenPaymentFragment.kt

```diff
@@ -190,8 +193,9 @@ class AdyenPaymentFragment : DaggerFragment(), AdyenPaymentView {
   }
 
   override fun finishCardConfiguration(
-      paymentMethod: com.adyen.checkout.base.model.paymentmethods.PaymentMethod,
-      isStored: Boolean, forget: Boolean, savedInstance: Bundle?) {
+    paymentMethod: com.adyen.checkout.base.model.paymentmethods.PaymentMethod,
```

Comment:
this com.adyen.checkout.base.model.paymentmethods.PaymentMethod could be imported to avoid having this (same happens on line 590